seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/master/JobHistoryService.java

```diff
@@ -248,7 +248,7 @@ public static final class JobState implements Serializable {
         private Long jobId;
         private String jobName;
         private JobStatus jobStatus;
-        private long submitTime;
+        private Long submitTime;
```

Comment:
Hi team, after this filed data type changed, the version upgrade will failed due to data type miss match.

Reply:
Hi @liunaijie ,thanks for your feedback, could you create an issue for this?

Reply:
https://github.com/apache/seatunnel/issues/6853